uint32 left
uint32 top
uint32 right
uint32 bottom
uint32 class_id
string class_name
